### ISSUE TITLE
Bootlooping after installing Evolution X 10.0

### ISSUE THREAD

**Issue by daiquansieucap** [mytja/treble_evo#86]
Hi guys. I just tried to install Evolution X 10.0 on my Samsung A30 (SM-A305F).
Although the installation was success, when I reboot my device, it kept bootlooping.
Is there a way to fix this issue?

**Comment by daiquansieucap** [comment]
> Could you please obtain `/proc/last_kmsg` after a bootloop from TWRP or other custom recovery?

Sorry to dissapoint, but I think that I can't upload that file for you (since it doesn't support the file type)

**Comment by mytja** [comment]
Just rename it to .txt and it'll let you upload it :)

**Comment by daiquansieucap** [comment]
> Just rename it to .txt and it'll let you upload it :)

[kmsg.txt](https://github.com/user-attachments/files/18088532/kmsg.txt)
[last_kmsg.txt](https://github.com/user-attachments/files/18088534/last_kmsg.txt)

**Comment by daiquansieucap** [comment]
> Just rename it to .txt and it'll let you upload it :)

Did you get that file?

**Comment by mytja** [comment]
Yes, thanks. (Un)fortunately it doesn't seem that anything in early boot/kernel is causing the bootloop, although your kernel is very old and long unsupported (Linux 4.4).

If I see correctly, adb is already running. I don't think I've included a patch that allows for adb to accept early requests by default, so I doubt `adb logcat` would work on current build during boot process, but you can at least try. I'll send another test build at the end of the week if logcat won't work.

Does latest A15 TrebleDroid or ponces' AOSP boot on your device? Did A14 boot? If not, I might not be able to help you, however I will at least try.

**Comment by daiquansieucap** [comment]
> Yes, thanks. (Un)fortunately it doesn't seem that anything in early boot/kernel is causing the bootloop, although your kernel is very old and long unsupported (Linux 4.4).
> 
> If I see correctly, adb is already running. I don't think I've included a patch that allows for adb to accept early requests by default, so I doubt `adb logcat` would work on current build during boot process, but you can at least try. I'll send another test build at the end of the week if logcat won't work.
> 
> Does latest A15 TrebleDroid or ponces' AOSP boot on your device? Did A14 boot? If not, I might not be able to help you, however I will at least try.

Did you mean, the GSI of Android 14?
If you ask the one that aren't a GSI file, I must say that works well
But if you ask me about the GSI rom, I don't know

**Comment by mytja** [comment]
Yeah, if TrebleDroid is bootlooping, this needs a fix from the TrebleDroid team. If you can capture `adb logcat` from TrebleDroid GSI during boot and send it to me, it would be appreciated.

**Comment by daiquansieucap** [comment]
> Yeah, if TrebleDroid is bootlooping, this needs a fix from the TrebleDroid team. If you can capture `adb logcat` from TrebleDroid GSI during boot and send it to me, it would be appreciated.

![Screenshot (1)](https://github.com/user-attachments/assets/b836c25f-124c-40de-b3b1-f4b996bda68e)
Stuck on that point

**Comment by daiquansieucap** [comment]
> Yeah, if TrebleDroid is bootlooping, this needs a fix from the TrebleDroid team. If you can capture `adb logcat` from TrebleDroid GSI during boot and send it to me, it would be appreciated.

![Screenshot (2)](https://github.com/user-attachments/assets/2e10a143-84d2-458f-ae9b-46f524c42566)
P/s: now it's running now (when the device is turn on)
